test/sdk-execution-tests/sdk_execution_tests.py

```diff
@@ -35,9 +41,24 @@
 CONFIG_PATH = os.path.join(PROJECT_ROOT, 'sdk', 'python', 'test_data',
                            'test_data_config.yaml')
 
+METADATA_HOST = '127.0.0.1'
+METADATA_PORT = 8080
+
 kfp_client = client.Client(host=KFP_ENDPOINT)
 kubernetes.config.load_kube_config()
 
+minio_client = Minio(
+    endpoint="127.0.0.1:9000",
+    access_key="minio",
+    secret_key="minio123",
+    secure=False,
+)
+
+BUCKET_NAME = "mlpipeline"
+
+print("Checking Minio connectivity...")
```

Comment:
Can you put this under `if __name__ == '__main__':`?

Reply:
this is not actually executed because we call the tests via `pytest`